x-pack/plugins/elastic_assistant/server/lib/langchain/graphs/default_assistant_graph/graph.ts

```diff
@@ -66,7 +82,16 @@ export const getDefaultAssistantGraph = ({
       },
       messages: {
         value: (x: BaseMessage[], y: BaseMessage[]) => x.concat(y),
-        default: () => messages,
+        default: () => [],
+      },
+      chatTitle: {
+        value: (x: string, y?: string) => y ?? x,
+        default: () => '',
+      },
+      conversation: {
+        value: (x: ConversationResponse | undefined, y?: ConversationResponse | undefined) =>
```

Comment:
nit: This had me do a double-take as I was like "Why is the conversation response being stored in state, wouldn't that be computed from `messages`?", then I realized it's just the current persisted state of the conversation. 

Maybe we should alias `ConversationResponse` to `Conversation` for more general use? I do like that we're aligning on the API response interface, that's nice.